js/server/modules/org/arangodb/foxx/request_context.js

```diff
@@ -455,7 +476,15 @@ extend(RequestContext.prototype, {
 /// The value of this object is either a single object or in the case of a multi
 /// element an array of objects.
 ///
-/// @EXAMPLES
+/// *Parameter*
+///
+///  * *paramName*: name of the body parameter in `req.parameters`.
+///  * *options*: a joi schema or an object with the following properties:
+///   * *description*: the documentation description of the request body.
+///   * *type*: the Foxx model or joi schema to use.
+///   * *allowInvalid* (optional): `true` if validation should be skipped. (Default: `false`)
```

Comment:
No idea. The functionality was already there, I just made it work the way the name implies ;)